docs/authoring/working_with_smes.md

```diff
@@ -0,0 +1,205 @@
+---
+id: working_with_smes
+title: Working with a subject expert
+sidebar_label: Working with a subject expert
+slug: /authoring/working_with_smes
+---
+
+## Getting the project started
+
+### Identify decision makers, team members, and sources of information
+
+Identify an internal and external team for the project. It should include:
+
+1. A project manager tasked with keeping the project on schedule. This can be either
+   internal or external.
+1. The developer or developers.
+1. An identified subject matter expert or experts.
+1. Key stakeholders who will need to sign off on the project or make decisions.
+
+This is also a good time to identify primary and secondary sources that the team
+can use to answer questions if an SME isn't readily available.
+
+### Schedule a check-in cadence
+
+Many projects require a check-in at a cadence of once a week. For slower projects,
+once every 2 weeks or once a month may be sufficient. Pick a day of the week
+and time in advance and set up a regular standing meeting on your calendar.
+
+Check-ins should generally be focused on quick status updates and blockers and take from 15 
+minutes to 1 hour, depending on their frequency.
+
+Not everyone needs to attend the check-in meetings.
+
+### Create a project communication system
+
+On the Assembly Line project, we default to using Kanban boards to track 
+interview progress. We use these columns:
+
+* Backlog, for cards that haven't been prioritized yet (they can represent ideas that need review)
+* To-do
+* Doing
+* Blocked, for cards that can't be acted on because of a "blocker" (which might include waiting for an SME's response)
+* Done
+
+You can use our [GitHub project template](https://github.com/orgs/SuffolkLITLab/projects/22/views/2) as a starting
+point for your project.
+
+Using the GitHub project, you can track progress, assign deadlines to tasks, and have threaded conversations (once the cards are
+converted into issues).
```

Comment:
I agree, but some of these instructions are built into the GitHub project template. [This one, for example.](https://github.com/orgs/SuffolkLITLab/projects/22?pane=issue&itemId=60119564) I would consider our public templates as part of our documentation and explain that anywhere we talk about our project management tools rather than duplicate documentation. We're more likely to update the templates and forget to update the documentation.

Reply:
Are you saying that GitHub itself provides this card or that our project has added it? I tend to err towards linking to original sources. That is, in this case I meant that we link to GitHub's own documentation on how to turn a card into an issue. I suspect that card, if it is one from GitHub, is only one of GitHub's sources of that information. If that is the case, I would be concerned that GitHub would change the card  setup in the future. I'd link to a GitHub documentation page that I had more confidence in.